File: trunk/Projects/Framework-Samples/NHibernate/VisualBasic/Sample.Data/ManagerObjects/OrderStatusManager.vb
Imports System
Imports System.Collections.Generic
Imports System.Text

Imports NHibernate
Imports Sample.Data.Generated.BusinessObjects
Imports Sample.Data.Generated.Base

Namespace Sample.Data.Generated.ManagerObjects
	'Public Partial Interface IOrderStatusManager
	'	Inherits IManagerBase(Of OrderStatus, System.String)
	'End Interface

	Partial Class OrderStatusManager
		Inherits ManagerBase(Of OrderStatus, System.String)
		Implements IOrderStatusManager
	End Class
End Namespace

File: Framework-Samples/NHibernate/CSharp/Sample.Data/ManagerObjects/OrderStatusManager.cs
using System;
using System.Collections.Generic;
using System.Text;

using NHibernate;
using Sample.Data.Generated.BusinessObjects;
using Sample.Data.Generated.Base;

namespace Sample.Data.Generated.ManagerObjects
{
    public partial interface IOrderStatusManager : IManagerBase<OrderStatus, string>
    {
	}
	
	partial class OrderStatusManager : ManagerBase<OrderStatus, string>, IOrderStatusManager
    {
	}
}